blueocean-pipeline-api-impl/src/main/java/io/jenkins/blueocean/rest/impl/pipeline/PipelineImpl.java

```diff
@@ -18,17 +20,18 @@
  */
 @Capability({JENKINS_WORKFLOW_JOB})
 public class PipelineImpl extends AbstractPipelineImpl {
-    protected PipelineImpl(Job job) {
-        super(job);
+    protected PipelineImpl(BlueOrganization organization, Job job) {
+        super(organization, job);
     }
 
     @Extension(ordinal = 1)
     public static class PipelineFactoryImpl extends BluePipelineFactory {
 
         @Override
         public BluePipeline getPipeline(Item item, Reachable parent) {
-            if (item instanceof WorkflowJob) {
-                return new PipelineImpl((Job) item);
+            BlueOrganization org = OrganizationFactory.getInstance().getContainingOrg(item);
```

Comment:
We should never be doing this lookup inside of the CTOR for the Pipeline Impls...